Azure-Samples/communication-services-ios-calling-hero
# Leah/share function

## Purpose

- As a user, I can share Group call ID through iOS Share Sheet
- Figma Design
<img src="https://user-images.githubusercontent.com/107075081/179143488-18179ae7-9be2-4c53-9534-3c689ce1694f.png"  width="200" />

- Implementation
<img src="https://user-images.githubusercontent.com/107075081/179143426-b2e21bab-635b-4d53-b0c9-76125dae2057.PNG" width="200" />

## Does this introduce a breaking change?

```
[x] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code
* Ensure you have set up AAD
* Go to invite screen and observe new design and share functionality

## What to Check
Verify that the following are valid
* share function properly shares Group call ID

## Other Information
